src/measure_theory/measure/outer_measure.lean

```diff
@@ -5,7 +5,7 @@ Authors: Johannes Hölzl, Mario Carneiro
 -/
 import analysis.specific_limits
 import measure_theory.pi_system
-import data.matrix.notation
+import data.fin.vec_notation
```

Comment:
Can you look through the other 13 files that import this and see if they can be relaxed too?